flamedisx/xenon/x1t_sr1.py

```diff
@@ -119,12 +119,12 @@ def cal_bias_tf(sig, fmap, domain_def, pivot_pt):
     bias_low = interpolate_tf(sig_tf, fmap[0], domain_def)
     bias_high = interpolate_tf(sig_tf, fmap[1], domain_def)
 
-    bias = (bias_high-bias_low)*pivot_pt + bias_low
+    bias = (bias_high - bias_low) * pivot_pt + bias_low
     bias_out = bias + tf.ones_like(bias)
 
     return bias_out
 
-def cal_rec_efficiency_tf(sig, fmap, domain_def, pivot_pt):
+def interpolate_reconstruction_efficiency(sig, fmap, domain_def, pivot_pt):
```

Comment:
Similar minor comment to above, it would be better to call it `calculate_reconstruction_efficiency` to reflect its functionality. It carries out some computation instead of just interpolating from the bias maps.